src/components/list/list-item.types.ts

```diff
@@ -48,7 +48,7 @@ export interface ListItem<T = any> {
 }
 
 export interface ListSeparator {
-    separator: true;
+    separator: boolean;
```

Comment:
Typing `separator` to `true` is intentional, as the property should either exist with a value of `true`, or not exist.

If you got some kind of error on this, I can help out trying to find some kind of fix 👍